src/FSharp.Charting.Gtk.fsproj

```diff
@@ -56,6 +56,26 @@
     <Reference Include="System.Core" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
+    <Reference Include="atk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\Program Files (x86)\GtkSharp\2.12\lib\gtk-sharp-2.0\atk-sharp.dll</HintPath>
```

Comment:
I'd really prefer if we pushed these DLLs as a nuget package and incorporated them that way (e.g. call it GtkSharp2), rather than via hardwired paths.  Do you think you could do that?  The nuget package specification could be included in this repo if you like.